docs/product/performance/requests.mdx

```diff
@@ -21,24 +21,15 @@ On the [**Requests** page](#requests-page), you get an overview of the domains t
   ></iframe>
 </div>
 
-## Prerequisites
+## Prerequisites and Limitations
 
 <Note>
 Network requests using non-HTTP protocols (FTP, WebSocket, etc.) are not supported at this time.
 </Note>
 
 Availability of HTTP request monitoring depends on the SDK your application uses. In most cases, Sentry's SDKs automatically enable HTTP request tracking. You can check your SDK's <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation documentation</PlatformLink> to see if it includes tracking HTTP requests.
 
```

Comment:
NIt, I might add a heading "Custom Instrumentation" to draw people's attention to this option.

Also, worth a discussion about whether we should suggest setting `network.protoco.name`, `network.protocol.version`, if people try to instrument an HTTP/2 connection?

Reply:
🤔 I can add this to our span data conventions!